docs/auth/tokens.rst

```diff
@@ -268,6 +268,126 @@ If you do not have the token UUID, but you do have the token value itself, you
 can use the :ref:`log-out` endpoint to delete it.
 
 
+Token Scoping: Domain Policies
+``````````````````````````````
+Tokens by default can be used to authorize any DNS operation on any domain
+within the user's account.
+Token Policies provide a means to restrict the scope of influence for a given
+API token.
+
+Using policies, the token's power is limited in two ways:
+
+1. the types of DNS operations that can be performed, such as :ref:`dynDNS
+   updates <update-api>` or :ref:`general RRset management <manage-rrsets>`.
+
+2. the set of domains on which these actions can be performed.
+
```

Comment:
say somewhere that tokens without policies are unrestricted

Reply:
or introduce the term "privileged tokens" here instead of further below

Reply:
done